app/frontend/src/components/forms/ExportSubmissions.vue

```diff
@@ -168,11 +168,16 @@ export default {
               .utc()
               .format()
             : undefined;
+
         const response = await formService.exportSubmissions(
           this.form.id,
-          from,
-          to,
-          this.exportFormat
+          this.exportFormat,
+          {
+            minDate: from,
+            maxDate: to,
+            // deleted: true,
```

Comment:
do we need these comments here for the final merge?

Reply:
not really. i left these comments in to show that they should be passed as true (boolean) if desired.
or not passed (or passed as undefined) if not desired.

Reply:
In this situation I think it's worth keeping the comments here as they'll provide breadcrumbs on how we can extend this call down the line to allow a user to choose these options.